### PR TITLE
Xeno jump cd buff

### PR DESCRIPTION
**Pull request by Lumipharon**
## About The Pull Request
Xeno jump CD changed from 2 seconds to 1 second (same as humans).
## Why It's Good For The Game
Makes xenos a bit more mobile, especially on the two most recent maps from CM that have a lot of close together platforms etc.
## Changelog
:cl:
balance: Reduced xeno jump CD from 2 seconds to 1 second
/:cl:

### CODE REVIEW

**Comment by iforgotmeword** [comment]
Wasn't the reason for xenos having a longer cooldown for jumping was because it didn't cost any stamina?

But meh what's would go wrong with this anyways...

**Comment by CVSthePharmacy** [comment]
WAY too op, keep it as is. There's a REASON why jump costs stamina

**Comment by chizzy376** [comment]
While I agree that having to jump 3 times in quick succession to traverse some parts of a map can feel ass, I think it's mostly a prevalent issue on the latest 2 CM ported maps.

If we instead made the platforms on kutjevo and lv non-dense we probably won't need to halve the cd of jumping for xenos.
It's a good qol for sure but the original cd was good for preventing xenos from using every possible window tile as an escape route
TL:DR: good PR but the issue lies in the maps

**Comment by SimplySmiley123** [comment]
Yeahhh there is a reason there's a CD on jump for xenos.
Buffing just because of CM maps ain't a good reason for enabling even easier, more often escapes.

**Comment by Lumipharon** [comment]
It has a 2 second CD because thats the number I pulled out of thin air when I added in jumping.

**Comment by Lefunny** [comment]
Good change tm xenos are meant to outmaneuver marines instead of being worst at it 👍

**Comment by SimplySmiley123** [comment]
> Good change tm xenos are meant to outmaneuver marines instead of being worst at it 👍

Didn't know they're parkour masters of mobility instead of murder machines.

**Comment by Lefunny** [comment]
> Didn't know they're parkour masters of mobility instead of murder machines.

Oh now you do! Pleasure to help with less experienced people in understanding what xenos can do, they can jump way higher than humans too and even walk in most surfaces, though i think you misunderstood something, might be a hard concept to understand but being a murder machine and also extremely high terrain movement relate.

**Comment by SimplySmiley123** [comment]
> > Didn't know they're parkour masters of mobility instead of murder machines.
> 
> Oh now you do! Pleasure to help with less experienced people in understanding what xenos can do, they can jump way higher than humans too and even walk in most surfaces, though i think you misunderstood something, might be a hard concept to understand but being a murder machine and also extremely high terrain movement relate.

It was sarcasm.
Xenos don't deal with stamina, have x-ray vision with night vision cuz why not and full map control.
Having slightly slower jump cooldown ain't the end of the world for you when you already have a piss easy time running around freely on the map.

**Comment by Lefunny** [comment]
> It was sarcasm. Xenos don't deal with stamina, have x-ray vision with night vision cuz why not and full map control. Having slightly slower jump cooldown ain't the end of the world for you when you already have a piss easy time running around freely on the map.

It was sarcasm. Xenos lose map control the moment marines land on ground, xenos having same jump time as marine is not the end of the world

+ Marines can move around the map just as easy with tad

**Comment by Victor239** [comment]
Makes sense, xenos are supposed to have superior agility but somehow get foiled by little window frames and shot to death waiting for CD.

**Comment by FosterGrime-Heymynameslouie** [comment]
sounds like a map issue than balance issue tbh

**Comment by chemistrymain3** [comment]
it would probably have it make more sense depending on the caste's actual mobility

a runner jumping a lot makes a lot of sense, the crusher leapfrogging around like a demented rabbit is less so

**Comment by Naaanii** [comment]
The hilarity of people conjuring up random facts from thin air is quite hilarious.

Please let this PR cook for a bit.
